pyramid_exclog/__init__.py

```diff
@@ -62,9 +65,14 @@ def _get_url(request):
 
 %(usr)s
 
-"""
+""")
 
 def _get_message(request):
+    """Return a string with useful information from the request.
+
+    On python 2 this method will return ``unicode`` and on Python 3 ``str``
+    will be returned. This seems to be what the logging module expects.
```

Comment:
I used the word "seems", because I was not sure. I still am not. I couldn't find it explicitly stated in the documentation or code.